source/Google/SignIn/ApiDefinition.cs

```diff
@@ -236,17 +215,17 @@ interface SignIn
 		[Export ("sharedInstance")]
 		SignIn SharedInstance { get; }
 
-		// -(BOOL)handleURL:(NSURL *)url sourceApplication:(NSString *)sourceApplication annotation:(id)annotation;
-		[Export ("handleURL:sourceApplication:annotation:")]
-		bool HandleUrl (NSUrl url, string sourceApplication, [NullAllowed] NSObject annotation);
+		// -(BOOL)handleURL:(NSURL *)url;
+		[Export ("handleURL:")]
+		bool HandleUrl (NSUrl url);
 
-		// -(BOOL)hasAuthInKeychain;
-		[Export ("hasAuthInKeychain")]
-		bool HasAuthInKeychain { get; }
+		// -(BOOL)hasPreviousSignIn;
+		[Export ("hasPreviousSignIn")]
+		bool HasPreviousSignIn { get; }
 
-		// -(void)signInSilently;
-		[Export ("signInSilently")]
-		void SignInUserSilently ();
+		// -(void)restorePreviousSignIn;
+		[Export ("restorePreviousSignIn")]
+		void RestorePreviousSignIn ();
```

Comment:
Should we keep `SignInUserSilently` (and obsolete it) and have it wrap this new member?

Reply:
Same comment as above, are we positive these two mean the same thing?